setup.py

```diff
@@ -31,29 +31,30 @@
         'apscheduler>=3.7,<3.8',
         'boto3<2.0',
         'cerberus>1.0,<2.0',
-        'django>=2.2,<2.3',
-        'djangorestframework>=3.9,<3.10',
-        'django-bootstrap4<1.2',
```

Comment:
A lot of these version changes aren't from me so I'm not super familiar

I added `setuptools-scm` and `uritemplate` as both are needed for our doc purposes.